scenes/test_menu.txt

```diff
@@ -0,0 +1,173 @@
+*label entry
+*params module log_buffer
+*temp test_count 0
+*temp tests_passed 0
+
+*comment Override choice UI with an automated system:
+*comment - cslib_choice_select: A list of of options to select sequentially
+*comment - cslib_choice_count: The number of 'visible' options at the end of the sequence
+*comment - cslib_choice_n: The 'visible' options at the end of the sequence
+*comment - cslib_ret: The last 'selected' option at the end of the sequence
+*set cslib_catch_choice true
+
+*comment :::::: SIMPLE tests ::::::
+*gosub test_start "simple - single item"
+*set cslib_choice_select "1"
+*gosub_scene cslib_menu build_simple "Hello"
+*gosub test_assert_equal 1 cslib_choice_count
+*gosub test_assert_equal 1 cslib_ret
+*gosub test_assert_equal "Hello" cslib_choice_1
+*gosub test_finish
+
+*gosub test_start "page-count invalid"
+*temp result_expected "Bug: cslib_menu._render_choice: this option should not be selectable"
+*set cslib_choice_select "1"
+*set cslib_catch_bug true
+*gosub_scene cslib_menu build_simple 1 2 3 4 5 6
+*set cslib_catch_bug false
```

Comment:
do you want to assert on the message?

Reply:
Oops, yes. Good catch :)

Reply:
I can't see the updated version with the fixed test